src/app/store/index.ts

```diff
@@ -1,20 +1,45 @@
 import { createStore, applyMiddleware, compose } from 'redux';
 import createSagaMiddleware from 'redux-saga';
 import logger from 'redux-logger';
+import { createEpicMiddleware } from 'redux-observable';
+
+import { itemsService } from '@core/services/items';
+import { MainComponentState } from '@core/pages/main/namespace/index';
+import { MainActions } from '@core/pages/main/store/actions/index';
 
 import rootReducer from './rootReducer';
 import rootSaga from './rootSaga';
+import { rootEpic } from './rootEpic';
+
+export interface RootStore {
+  main: MainComponentState;
+}
+
+const dependencies = {
+  itemsService,
+};
+export type Dependencies = typeof dependencies;
+
+const epicMiddleware = createEpicMiddleware<
+  MainActions,
+  MainActions,
```

Comment:
duplicated entries? is that on purpose?

Reply:
Yes, maybe it can be done other way but that's how I got it working. 
To my knowledge createEpicMiddleware generic parameters are `<InputActions, OutputActions, State, Dependencies>`, that's why entries are duplicated.